Index: src/pages/_app.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react'\nimport App from 'next/app'\n\nimport 'antd/dist/antd.less'\n\nimport SiteLayout from '../ui/components/layout/Site'\nimport SettingsLayout from '../ui/components/layout/Settings'\n\nclass MyApp extends App {\n  render () {\n    const { Component, pageProps, router } = this.props\n\n    if (router.pathname.startsWith('/settings')) {\n      return (\n        <SiteLayout>\n          <SettingsLayout>\n            <Component {...pageProps} />\n          </SettingsLayout>\n        </SiteLayout>\n      )\n    }\n\n    return (\n      <SiteLayout>\n        <Component {...pageProps} />\n      </SiteLayout>\n    )\n  }\n}\n\nexport default MyApp\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/pages/_app.js	(revision 697c0591d3def1c62ddf2a57dfce8ca136862487)
+++ src/pages/_app.js	(date 1594128466295)
@@ -1,31 +1,31 @@
-import React from 'react'
-import App from 'next/app'
+import React from "react";
+import App from "next/app";
 
-import 'antd/dist/antd.less'
+import "antd/dist/antd.less";
 
-import SiteLayout from '../ui/components/layout/Site'
-import SettingsLayout from '../ui/components/layout/Settings'
+import SiteLayout from "../components/layout/Site";
+import SettingsLayout from "../components/layout/Settings";
 
 class MyApp extends App {
-  render () {
-    const { Component, pageProps, router } = this.props
+  render() {
+    const { Component, pageProps, router } = this.props;
 
-    if (router.pathname.startsWith('/settings')) {
+    if (router.pathname.startsWith("/settings")) {
       return (
         <SiteLayout>
           <SettingsLayout>
             <Component {...pageProps} />
           </SettingsLayout>
         </SiteLayout>
-      )
+      );
     }
 
     return (
       <SiteLayout>
         <Component {...pageProps} />
       </SiteLayout>
-    )
+    );
   }
 }
 
-export default MyApp
+export default MyApp;
Index: src/pages/customer.ts
===================================================================
--- src/pages/customer.ts	(revision 697c0591d3def1c62ddf2a57dfce8ca136862487)
+++ src/pages/customer.ts	(revision 697c0591d3def1c62ddf2a57dfce8ca136862487)
@@ -1,5 +0,0 @@
-
-const Customer = () => ('Customer')
-
-
-export default Customer
